src/index.css

```diff
@@ -1,9 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 body {
+  margin: 20px;
   text-align: center;
+  align-items: center;
+  margin-left: 12rem;
+  background-color: rgb(204, 209, 211);
+}
+
+.navigation-bar {
+  display: flex;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  background-color: #fff;
+  border-bottom: 1px solid gray;
+  margin-left: 4rem;
+  margin-top: 1.5rem;
+}
+
+.main-heading {
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: rgba(6, 86, 233, 0.658);
+  padding-top: 1rem;
+  padding-left: 2rem;
+}
+
```

Comment:
- [ ] As the previous reviewer said, you should follow the [Bookstore template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). You might want to consider working with the right font-families from the Template. 😃 
- RobotoSlab font-family
- Montserrat font-family

Reply:
- [ ] As the previous reviewer suggested, you should try to match as much as possible the template given by Microverse. Including setting the correct font families and colors. As you can see in the screenshot below (at the bottom), some of the fonts are still not matching Montserrat or RobotoSlab.

![image](https://user-images.githubusercontent.com/79114233/142716508-bc09cde7-bf71-434d-9100-31d06b7fbff2.png)

Hint: You can get both of these fonts for free in [fonts.google.com
](https://fonts.google.com/)